baseframe/filters.py

```diff
@@ -54,6 +54,19 @@ def age(dt):
     else:
         return _("%(num)s years ago", num=int(delta.days / 365))
 
+@baseframe.app_template_filter('initials')
+def initials(field):
+    """
+    Return first and last initials from the given input, meant for use as avatar stand-in.
+    """
+    parts = field.split()
+    if len(parts) > 1:
+        return parts[0][0] + parts[-1][0]
+    elif parts[0]:
+        return parts[0][0]
+    else:
+        return ''
```

Comment:
Need a test that hits this line. Inputs: `None`, `''` (empty string) and `' '` (space). `None` will break this filter, so your very first line has to be `if not field: return ''`